app/helpers/pieces_helper.rb

```diff
@@ -1,2 +1,99 @@
 module PiecesHelper
+  def can_capture?(new_x, new_y)
```

Comment:
I think we should rename this to just `capture`.  `can_capture?` sounds like it just returns a boolean if it is able to capture, but it has the unexpected side effect of actually performing the capture and updating the piece position. If we say `capture` we know we are trying to capture, but simply return false if we can't.